src/Storages/System/InformationSchema/schemata.sql

```diff
@@ -1,26 +1,33 @@
 ATTACH VIEW schemata
-(
-    `catalog_name` String,
-    `schema_name` String,
-    `schema_owner` String,
-    `default_character_set_catalog` Nullable(String),
-    `default_character_set_schema` Nullable(String),
-    `default_character_set_name` Nullable(String),
-    `sql_path` Nullable(String),
-    `CATALOG_NAME` String ALIAS catalog_name,
-    `SCHEMA_NAME` String ALIAS schema_name,
-    `SCHEMA_OWNER` String ALIAS schema_owner,
-    `DEFAULT_CHARACTER_SET_CATALOG` Nullable(String) ALIAS default_character_set_catalog,
-    `DEFAULT_CHARACTER_SET_SCHEMA` Nullable(String) ALIAS default_character_set_schema,
-    `DEFAULT_CHARACTER_SET_NAME` Nullable(String) ALIAS default_character_set_name,
-    `SQL_PATH` Nullable(String) ALIAS sql_path
-) AS
-SELECT
-    name AS catalog_name,
-    name AS schema_name,
-    'default' AS schema_owner,
-    NULL AS default_character_set_catalog,
-    NULL AS default_character_set_schema,
-    NULL AS default_character_set_name,
-    NULL AS sql_path
+    (
```

Comment:
Thoughts about the changes in `information_schema`:
- even if needed by Tableau Online, they are technically independent of the binary protocol implementation, and should be pushed as a separate PR
- [Our docs](https://clickhouse.com/docs/en/operations/system-tables/information_schema) do not mention that columns in `information_schema.schemata` can be accessed both in a lower-case and an upper-case form (`catalog_name` vs. `CATALOG_NAME`). It has been like that before your PR, but now is a good time to correct this.
- Upper-case name were previously implemented using alias columns ([docs](https://clickhouse.com/docs/en/sql-reference/statements/create/table#alias)). This had the specific effect that `SELECT *` excluded them, which is the behavior one would intuitively expect. This PR makes `SELECT *` show every column twice - unbeautiful, to be honest. What was the reason for that? Does Tableau Online execute `SELECT *` and expect uppercase column names? A less invasive alternative would be to retain the alias expressions in the view definition but somehow teach Tableau Online to apply setting `asteriks_include_alias_columns`. This will achieve the same behavior as this PR does.

Reply:
I will create a new PR for information_schema changes. I reverted the changes for now.